packages/studio/src/utils/PageDataValidator.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * PageDataValidator contains various static utility methods
+ * for validation of user-inputted page data.
+ */
+export default class PageDataValidator {
+  /**
+   * Throws an error if the user-inputted page data is invalid.
+   */
+  validate(
```

Comment:
For better usability, I don't think we should throw an Error on the first problem we encounter. Suppose a user has both a bad page name and a bad url. They'd first fix one error, try and confirm, then get a new, second error pop up. If we show them all the issues at once, they can fix them all at one.

Reply:
Updated! I also thought the error messages were a bit crowded so I shortened them a bit: 

<img width="300" alt="Screen Shot 2023-08-07 at 3 25 04 PM" src="https://github.com/yext/studio/assets/78767440/b82df2a7-0e87-4dfe-ba13-e664ebc31542">

Reply:
To clarify, the error messages used to include the actual page name in the error message and have "error adding page:" in front, but I removed these. LMK if we should add these back.